DiffAugment-stylegan2-pytorch/train.py

```diff
@@ -268,6 +269,13 @@ def setup_training_loop_kwargs(
         assert isinstance(aug, str)
         desc += f'-{aug}'
 
+
+    if diffaugment_placement is None and diffaugment is not None:
```

Comment:
Probably wold be better with
`    if diffaugment_placement is None and diffaugment:
`